### DISCUSSION TITLE
Is there a changelog?

### DISCUSSION THREAD

**Discussion by Hornwitser** [transmission/transmission#5880] [Q&A] [answered]
I see that there's a version 4 now and I'm running version 3, but what is the difference? What's new? Why should I update? I can't find any place with a simple list of new features, bugfixes and so on.

**Comment by NiklasBr** [comment, marked as answer]
https://github.com/transmission/transmission/releases has all your changes listed for each new version.

**Reply by Hornwitser** [reply]
Thank you! I wasn't able to find this despite searching for it, maybe you should mention it in the README.
